Refactor the focusedColumnIndex description for 19-2 (#175)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/GridBase/1 Configuration/focusedColumnIndex.md

---
---
##### shortDescription
The index of the column that contains the focused data cell.

---
This index changes when users do the following:  

- Navigate through cells    
- [Reorder](/concepts/05%20Widgets/DataGrid/15%20Columns/25%20Column%20Reordering '/Documentation/Guide/Widgets/{WidgetName}/Columns/Column_Reordering/') the columns    
- [Hide](/api-reference/_hidden/GridBaseColumn/allowHiding.md '{basewidgetpath}/Configuration/columns/#allowHiding') the columns 

The following image illustrates the indexing system: 

![DevExtreme HTML5 JavaScript {WidgetName} Column Index](/images/{WidgetName}/visual_elements/columnIndexes.png)

> [Band columns](/api-reference/_hidden/GridBaseColumn/isBand.md '{basewidgetpath}/Configuration/columns/#isBand') cannot be focused since they do not have indexes. However, users can focus banded columns.

Reviewer: ```suggestion
[Band columns](/api-reference/_hidden/GridBaseColumn/isBand.md '{basewidgetpath}/Configuration/columns/#isBand') do not have indexes and cannot be focused. However, users can focus banded columns.
```
